po/zh_CN.po

```diff
@@ -633,268 +658,264 @@ msgstr "%ls: 未知信号 '%ls'"
-msgstr ""
+msgstr "%ls: 预期收到 <= %d 个参数；实际收到 %d 个\n"
 
 #, c-format
 msgid "%ls: expected >= %d arguments; got %d\n"
-msgstr ""
+msgstr "%ls: 预期收到 >= %d 个参数；实际收到 %d 个\n"
 
 #, c-format
 msgid "%ls: expected a numeric value"
-msgstr "%ls: 预期数字类型值"
+msgstr "%ls: 预期收到数值"
 
-#
 #, c-format
 msgid "%ls: fish was not built with embedded files"
-msgstr ""
+msgstr "%ls: fish 构建时未包含嵌入文件"
 
 #, c-format
 msgid "%ls: given %d indexes but %d values\n"
-msgstr "%ls: 给定的 %d 索引但 %d 值\n"
+msgstr "%ls: 给定索引 %d 但只有 %d 个值\n"
 
 #, c-format
 msgid "%ls: invalid option combination\n"
-msgstr "%ls: 选项组合无效\n"
+msgstr "%ls: 无效的选项组合\n"
 
 #, c-format
 msgid "%ls: invalid option combination, %ls\n"
-msgstr ""
+msgstr "%ls: 无效的选项组合，%ls\n"
 
 #, c-format
 msgid "%ls: invalid underline style: %ls\n"
-msgstr ""
+msgstr "%ls: 无效的下划线样式：%ls\n"
 
 #, c-format
 msgid "%ls: job %d ('%ls') was stopped and has been signalled to continue.\n"
-msgstr "%ls: 任务 %d ('%ls') 已停止并指示要继续 .\n"
+msgstr "%ls: 作业 %d ('%ls') 已停止并收到继续信号。\n"
 
 #, c-format
 msgid "%ls: line/column index starts at 1"
-msgstr ""
+msgstr "%ls: 行/列索引从1开始"
 
 #, c-format
 msgid "%ls: missing argument\n"
-msgstr ""
+msgstr "%ls: 缺少参数\n"
 
 #, c-format
 msgid "%ls: missing filename argument or input redirection\n"
-msgstr ""
+msgstr "%ls: 缺少文件名参数或输入重定向\n"
 
 #, c-format
 msgid "%ls: missing subcommand\n"
 msgstr "%ls: 缺少子命令\n"
 
 #, c-format
 msgid "%ls: nothing to choose from\n"
-msgstr "%ls: 无选择于 \n"
+msgstr "%ls: 没有可选项\n"
 
 #, c-format
 msgid "%ls: realpath failed: %s\n"
-msgstr "%ls: realath 失败:  %s\n"
+msgstr "%ls: realpath 失败：%s\n"
 
 #, c-format
 msgid "%ls: scope can be only one of: universal function global local\n"
-msgstr "%ls: 范围只能是: 通用函数全局本地\n"
+msgstr "%ls: 范围只能是：universal（通用）function（函数）global（全局）local（局域）\n"
 
 #, c-format
 msgid "%ls: stdin is closed\n"
 msgstr "%ls: stdin 已关闭\n"
 
 #, c-format
 msgid "%ls: successfully set universal '%ls'; but a global by that name shadows it\n"
-msgstr "%ls: 成功设置通用 '%ls' ; 但是这个名称的全局设置覆盖它\n"
+msgstr "%ls: 成功设置通用变量 '%ls' ；但是同名全局变量遮蔽它\n"
 
 #, c-format
 msgid "%ls: the -k/--key syntax is no longer supported. See `bind --help` and `bind --key-names`\n"
-msgstr ""
+msgstr "%ls：-k/--key 语法不再受支持。参见 `bind --help` 和 `bind --key-names`\n"
 
 #, c-format
 msgid "%ls: there is no line %ls\n"
-msgstr ""
+msgstr "%ls: 不存在行 %ls\n"
 
 #, c-format
 msgid "%ls: too many arguments\n"
 msgstr "%ls: 参数太多\n"
 
 #, c-format
 msgid "%ls: usage of -i for --silent is deprecated. Please use -s or --silent instead.\n"
-msgstr "%ls: 对-i的用法 --silent 已贬值. 请使用 - s 或 --silent 代替.\n"
+msgstr "%ls: 参数 -i 用于表示 --silent 已弃用。请改用 -s 或 --silent 代替。\n"
 
 #, c-format
 msgid "%ls: value not completely converted (can't convert '%ls')"
-msgstr "%ls: 数值未完全转换( 无法转换 '%ls' )"
+msgstr "%ls: 数值未完全转换 (无法转换 '%ls')"
 
 #, c-format
 msgid "%ls: variable '%ls' is read-only\n"
-msgstr ""
+msgstr "%ls: 变量 '%ls' 只读\n"
 
 #, c-format
 msgid "%lsand %lu more rows"
-msgstr "%lsand %lu 更多行数"
+msgstr "%ls还有 %lu 行"
```

Comment:
```suggestion
msgstr "%ls 还有 %lu 行"
```

Reply:
the first placeholder will be an ellipsis (`…`) which works well without space in other languages

Reply:
It works that way in Chinese too.